scripts/automator/packages/clang-brew

```diff
@@ -0,0 +1,3 @@
+# Brew does not supply "clang", so exclude it here
+compiler=""
+
```

Comment:
Should be doable.  The command line argument is dynamically generated from this, so it will become --cxx (variable), which ahould work. Will update.

Reply:
I started down this path, switching the script to take in `--cxx <variable>` and eliminating the `--version-postfix` as well, so users can do stuff like:

```
--cxx gcc9
--cxx clang++-7
--cxx g++-mp-9
--cxx clang-10
```

In my prior code, I would have hardcoded knowledge about gcc and clang, and would have branches for each.  Now, the 'automator' has no knowledge, and it just defers to the config tree.  So for the build tree, we have these files. 

So when the user types `--compiler gcc`, then the value gets sourced as a file here (and layered up with the OS and architecture too). 

```
clang-darwin_x86_64
clang-defaults
clang-linux_x86_64
clang-msys_nt_x86_64
compiler-clang
compiler-defaults
compiler-gcc
gcc-darwin_x86_64
gcc-defaults
gcc-linux_x86_64
gcc-msys_nt_x86_64
machine-x86_64
os-darwin
os-defaults
os-linux
os-msys_nt
```

So the first problem is confusion between gcc, g++, clang, clang++.  I can no longer take the variable at face value, and instead either need files for all of these, or I need to add code to convert user input to either `gcc` or `clang`. 

Second, is figuring out how to strip-off the postfix from something like `--cxx gcc9`.  I need to strip off the postfix because otherwise it won't match a filename. 

The end result is adding a lot more complexity than any tangible benefit. 

For the next go round with build scripts, let's figure out the desired interface together that still lets us support and scale the back-end with data.  You might see or have other ways to do this that I'm just not getting right now :-)

Reply:
Unfortunately have to defer this fix to the next re-write of the build script.